packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -517,7 +520,8 @@ export default class Editor {
     ): boolean;
 
     public select(arg1: any, arg2?: any, arg3?: any, arg4?: any): boolean {

```

Comment:
Is this a duplicate of the const defined in selectRange?

Reply:
This is the public editor API, while selectRange is a CoreAPI and can't be called outside editor.